src/index.js

```diff
@@ -14,7 +13,9 @@ ReactDOM.render(
   <BrowserRouter>
     <VideoContextProvider>
       <FilterContextProvider>
-        <App />
+        <AuthContextProvider>
+          <App />
+        </AuthContextProvider>
       </FilterContextProvider>
     </VideoContextProvider>
```

Comment:
can define a single context provider to wrap up all other contexts & directly import & use here.